front-end/src/components/Admin/gridEntities.js

```diff
@@ -6,6 +6,16 @@ import contracts from '../../SampleJson/contracts.json';
 import organizations from '../../SampleJson/organizations.json';
 import clients from '../../SampleJson/clients.json';
 
+
+var printIcon = function (cell) {
+  console.log(cell.getValue());
+  if (cell.getValue() === 'Male') {
+    return '<i class="fas fa-male"/>';
```

Comment:
Please check existing code (search project for <FontAwesomeIcon) to learn how to insert fontawesome icons.